### ISSUE TITLE
I can't change my nextcloud logo !

### ISSUE THREAD

**Issue by chiyi4488** [nextcloud/server#9274]
### Steps to reproduce
1. Open settings, set theme.
2. Press upload logo
3. Then nothing happened (upload success)

### Expected behaviour
Logo update

### Actual behaviour
Logo not updated

### Server configuration

**Operating system**:
nextcloudpi (Debian 9 stretch)
**Web server:**
Apache 2.4.25
**Database:**
mysql 10.1.23
**PHP version:**
7.0
**Nextcloud version:** (see Nextcloud admin page)
13.0.1
**Updated from an older Nextcloud/ownCloud or fresh install:**

**Where did you install Nextcloud from:**

**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
No errors have been found.
</details>

**List of activated apps:**
<details>
<summary>App list</summary>
  - activity: 2.6.1
  - admin_notifications: 1.0.1
  - announcementcenter: 3.2.1
  - apporder: 0.4.1
  - beame_insta_ssl: 0.0.6
  - bruteforcesettings: 1.0.3
  - calendar: 1.6.1
  - caniupdate: 0.1.2
  - comments: 1.3.0
  - dav: 1.4.6
  - defaultlinkopen: 1.0.0
  - drop_account: 0.0.7
  - federatedfilesharing: 1.3.1
  - federation: 1.3.0
  - files: 1.8.0
  - files_accesscontrol: 1.3.0
  - files_downloadactivity: 1.2.0
  - files_pdfviewer: 1.2.1
  - files_rightclick: 0.5.0
  - files_sharing: 1.5.0
  - files_snapshots: 0.1.1
  - files_texteditor: 2.5.1
  - files_trashbin: 1.3.0
  - files_versions: 1.6.0
  - files_videoplayer: 1.2.0
  - firstrunwizard: 2.2.1
  - flowupload: 0.0.7
  - gallery: 18.0.0
  - groupfolders: 1.2.0
  - impersonate: 1.0.4
  - logreader: 2.0.0
  - lookup_server_connector: 1.1.0
  - mail: 0.7.10
  - metadata: 0.6.0
  - nextcloud_announcements: 1.2.0
  - notifications: 2.1.2
  - oauth2: 1.1.0
  - password_policy: 1.3.0
  - provisioning_api: 1.3.0
  - quota_warning: 1.2.0
  - ransomware_protection: 1.1.0
  - serverinfo: 1.3.0
  - sharebymail: 1.3.0
  - sociallogin: 1.3.4
  - socialsharing_email: 1.0.3
  - socialsharing_facebook: 1.0.2
  - socialsharing_googleplus: 1.0.2
  - survey_client: 1.1.0
  - systemtags: 1.3.0
  - theming: 1.4.1
  - twofactor_backupcodes: 1.2.3
  - updatenotification: 1.3.0
  - user_usage_report: 1.0.1
  - workflowengine: 1.3.0
Disabled:
  - admin_audit
  - checksum
  - encryption
  - files_antivirus
  - files_external
  - files_inotify
  - registration
  - theming_customcss
  - user_external
  - user_ldap

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>
{
    "system": {
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": {
            "0": "localhost",
            "5": "nextcloudpi.local",
            "1": "192.168.1.103",
            "4": "cloud.chichien.art",
            "2": "cloud.chichien.art"
        },
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/cloud.chichien.art",
        "dbtype": "mysql",
        "version": "13.0.1.1",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "timeout": 0
        },
        "mail_smtpmode": "php",
        "mail_smtpauthtype": "LOGIN",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "overwriteprotocol": "https",
        "maintenance": false,
        "loglevel": 0,
        "log_type": "file"
    }
}

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

### Client configuration
**Browser:*chrome*

**Operating system:*win7*

**Comment by juliushaertl** [comment]
Duplicate of #8888. Fix will be included in 13.0.2

**Comment by chiyi4488** [comment]
OK, thank you.
